multiplayer/src/components/ArcadeSimulator.tsx

```diff
@@ -173,7 +173,7 @@ export default function Render() {
                 "tw-h-[calc(100vh-16rem)] tw-w-screen md:tw-w-[calc(100vw-6rem)]"
             }
             style={{
-                filter: gamePaused ? "grayscale(1)" : "none",
+                filter: gamePaused ? "grayscale(60%) blur(8px)" : "none",
```

Comment:
Can we maybe try like half this? 8px is pretty intense, my eyes hurt a little looking at the screenshot

Reply:
brought it down to 3px. Maybe this is better?
![image](https://user-images.githubusercontent.com/12176807/200234720-9a5cd8b7-fb25-42ee-89c7-c5e2ac9a116e.png)

Reply:
Actually, let's go with 1px.
![image](https://user-images.githubusercontent.com/12176807/200235215-f201c6cb-2935-4a87-be8b-f9363a5538b1.png)

Reply:
Probably not feasible, but going to the opposite extreme gives a nice effect. Here's a blur of 60px:
![image](https://user-images.githubusercontent.com/12176807/200235627-d7ba3796-c3a9-4e47-85cc-91648bdea9fd.png)

Reply:
1 and 3 both look fine to me,